date: '2021-12-13'
sections:
  security_fixes:
    - '{% octicon "alert" aria-label="O ícone de alerta" %} **Crítico:** Uma vulnerabilidade de execução de código remoto na biblioteca Log4j, identificada como [CVE-2021-44228](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44228), afetou todas as versões de {% data variables.product.prodname_ghe_server %} anteriores à 3.3.1. A biblioteca Log4j é usada em um serviço de código aberto sendo executado na instância {% data variables.product.prodname_ghe_server %}. Esta vulnerabilidade foi corrigida nas versões {% data variables.product.prodname_ghe_server %} 3.0.22, 3.1.14, 3.2.6 e 3.3.1.  Confira mais informações em [esta postagem](https://github.blog/2021-12-13-githubs-response-to-log4j-vulnerability-cve-2021-44228/) no blog do GitHub.'
    - '**Atualização de 17 de dezembro de 2021**: as correções nesta versão também mitigam [CVE-2021-45046](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046) que foi publicada após esta versão. Não é necessária nenhuma atualização adicional para {% data variables.product.prodname_ghe_server %} para mitigar tanto CVE-2021-44228 como CVE-2021-45046.'
  known_issues:
    - "Depois da atualização para o {% data variables.product.prodname_ghe_server %} 3.3, as {% data variables.product.prodname_actions %} podem falhar na inicialização automática. Para resolver esse problema, conecte-se ao dispositivo via SSH e execute o comando \"ghe-actions-start\"."
    - "Em uma instância de {% data variables.product.prodname_ghe_server %} recém-configurada sem usuários, um invasor pode criar o primeiro usuário administrador."
    - As regras de firewall personalizadas são removidas durante o processo de atualização.
    - "Arquivos LFS do Git [enviados através da interface Web](https://github.com/blog/2105-upload-files-to-your-repositories) são adicionados diretamente ao repositório e de maneira incorreta."
    - "Os problemas não podem ser fechados se contiverem um permalink para um blob no mesmo repositório, em que o caminho do arquivo de blob é maior que 255 caracteres."
    - "Quando a opção \"Usuários podem pesquisar pelo GitHub.com\" está habilitada com o GitHub Connect, os problemas em repositórios privados e internos não são incluídos nos resultados de pesquisa do GitHub.com."
    - "O registro npm {% data variables.product.prodname_registry %} não retorna mais um valor temporal em respostas de metadados. Isso foi feito para permitir melhorias substanciais de desempenho. Continuamos a ter todos os dados necessários para retornar um valor temporal como parte da resposta de metadados e continuaremos a retornar este valor no futuro quando tivermos resolvido os problemas de desempenho existentes."
    - Os limites de recursos que são específicos para processamento de ganchos de pré-recebimento podem causar falha em alguns ganchos de pré-recebimento.
    - 'As configurações de armazenamento de {% data variables.product.prodname_actions %} não podem ser validadas e salvas no {% data variables.enterprise.management_console %} quando a opção "Forçar estilo de caminho" estiver selecionada e deverão ser definidas com a ferramenta de linha de comando "ghe-actions-precheck".'    
    - 'As instâncias do {% data variables.product.prodname_ghe_server %} 3.3 instaladas no Azure e provisionadas com mais de 32 núcleos de CPU falhariam na inicialização devido a um bug presente no kernel do Linux atual. [Atualizado em: 08-04-2022]'
    - '{% data reusables.release-notes.ghas-3.4-secret-scanning-known-issue %}'
